pybleau/app/plotting/plot_style.py

```diff
@@ -302,23 +302,23 @@ def _get_plot_mappers(self, plot, x_mapper=None, y_mapper=None,
         missing_mapper_msg = "The plot is missing the attribute {}. Please " \
                              "provide the mapper explicitly"
         if x_mapper is None:
-            if hasattr(plot, "x_axis"):
+            if plot.x_axis:
                 x_mapper = plot.x_axis.mapper
             else:
                 msg = missing_mapper_msg.format("x_axis")
                 logger.exception(msg)
                 raise ValueError(msg)
 
         if y_mapper is None:
-            if hasattr(plot, "y_axis"):
+            if plot.y_axis:
                 y_mapper = plot.y_axis.mapper
             else:
                 msg = missing_mapper_msg.format("y_axis")
                 logger.exception(msg)
                 raise ValueError(msg)
 
         if second_y_mapper is None:
-            if hasattr(plot, "second_y_axis"):
+            if hasattr(plot, "second_y_axis") and plot.second_y_axis:
```

Comment:
Would it be just as good to check if `plot.second_y_axis` is not `None`?

Reply:
Great question! But the answer is "no" because that function is designed to also work with normal chaco Plot instances, not just ours which now always has that attribute.